tests/TestCase.php

```diff
@@ -178,7 +182,7 @@ protected function assertSameResponse($firstResponse, $secondResponse)
      */
     protected function assertDifferentResponse($firstResponse, $secondResponse)
     {
-        self::assertThat($firstResponse->getContent() != $secondResponse->getContent(), self::isTrue(), 'Failed to assert that two response are the same');
+        self::assertThat($firstResponse->getContent() != $secondResponse->getContent(), self::isTrue(), 'Failed to assert that two response are different');
```

Comment:
Can we do a strict check here? `!== `

Reply:
Then `assertSameResponse` should have a strict check as well, no?